SpringProject/src/main/java/com/gradproject2019/conferences/exception/ConferenceConflictException.java

```diff
@@ -0,0 +1,8 @@
+package com.gradproject2019.conferences.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.CONFLICT, reason="Conference already exists.")
+public class ConferenceConflictException extends RuntimeException {
```

Comment:
ConferenceConflict doesn't really tell you what's happened here ConferenceAlreadyExists is better.